### DISCUSSION TITLE
Create ZSH Alias for NPM Global packages

### DISCUSSION THREAD

**Discussion by jrock2004** [nicknisi/dotfiles#196] [Ideas]
So, since I gotten more into LSP servers with Neovim, I decided to create an alias which will let me easily update all those npm packages that I need to keep updated. Thoughts if there is maybe a better way?

```zsh
alias npmpackages='volta install bash-language-server diagnostic-languageserver dockerfile-language-server-nodejs @lifeart/ember-language-server vscode-langservers-extracted typescript typescript-language-server @tailwindcss/language-server vim-language-server yaml-language-server ember-cli'
```

**Comment by nicknisi** [comment]
This isn't a bad idea 🤔 

What I am doing right now is trying to let Neovim manage it via a plugin - [nvim-lsp-installer](https://github.com/williamboman/nvim-lsp-installer). With this plugin, you can type `:LspInstall` with a TS file open, for example, and get a popup like this:

![capture-20220203061839](https://user-images.githubusercontent.com/293805/152341522-580ae2f4-ee1c-4b64-9ac1-204d90997e3c.png)
